src/NHapi.Base/Parser/DefaultXMLParser.cs

```diff
@@ -63,30 +60,35 @@ public static new void Main(string[] args)
                 var messageFile = new FileInfo(args[0]);
                 var fileLength = SupportClass.FileLength(messageFile);
```

Comment:
Can `SupportClass.FileLength` be changed to return the type required below (`var buffer = new char[(int)fileLength];`) to avoid the need to cast?

Reply:
I'll have a look.